Parallel build error: LaTeX Error: File `fig/idft_flow/schemes_all' not found

When testing 1.5_rc1, using parallel build (-j6), the following error showed up:
```
LaTeX Error: File `fig/idft_flow/schemes_all' not found
...
BUILDSTDERR:   /builddir/build/BUILD/votca-1.5_rc1/xtp/manual/CMakeModules/UseLATEX.cmake:1824 (latex_execute_latex)
BUILDSTDERR: make[2]: *** [xtp/manual/CMakeFiles/xtp-manual_dvi.dir/build.make:122: xtp/manual/xtp-manual.dvi] Error 1
make[2]: Leaving directory '/builddir/build/BUILD/votca-1.5_rc1/x86_64-redhat-linux-gnu'
BUILDSTDERR: make[1]: *** [CMakeFiles/Makefile2:6967: xtp/manual/CMakeFiles/xtp-manual_dvi.dir/all] Error 2
BUILDSTDERR: make[1]: *** Waiting for unfinished jobs....
```

--- a/manual/fig/idft_flow/CMakeLists.txt
+++ b/manual/fig/idft_flow/CMakeLists.txt
@@ -1,9 +1,13 @@
-foreach(PIC scheme schemes_all) 
+foreach(PIC scheme schemes_all)
+  #Run FIGTEX2EPS in ${PIC} subdir to avoid overwriting on latex tmp files in parallel build
+  file(MAKE_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/${CURRENT_DIR}/${PIC})
   add_custom_command(OUTPUT ${CURRENT_DIR}/${PIC}.eps
     COMMAND ${CMAKE_COMMAND} -E copy_if_different ${CMAKE_CURRENT_SOURCE_DIR}/${CURRENT_DIR}/${PIC}.fig
-      ${CMAKE_CURRENT_BINARY_DIR}/${CURRENT_DIR}/${PIC}.fig
-    COMMAND ${FIGTEX2EPS} ${CMAKE_CURRENT_BINARY_DIR}/${CURRENT_DIR}/${PIC}.fig
-      ${CMAKE_CURRENT_BINARY_DIR}/${CURRENT_DIR}/${PIC}.eps
-    WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/${CURRENT_DIR}
+       ${CMAKE_CURRENT_BINARY_DIR}/${CURRENT_DIR}/${PIC}/${PIC}.fig
+    COMMAND ${FIGTEX2EPS} -v ${CMAKE_CURRENT_BINARY_DIR}/${CURRENT_DIR}/${PIC}/${PIC}.fig
+       ${CMAKE_CURRENT_BINARY_DIR}/${CURRENT_DIR}/${PIC}/${PIC}.eps
+    COMMAND ${CMAKE_COMMAND} -E copy_if_different ${CMAKE_CURRENT_BINARY_DIR}/${CURRENT_DIR}/${PIC}/${PIC}.eps
+       ${CMAKE_CURRENT_BINARY_DIR}/${CURRENT_DIR}/${PIC}.eps
+    WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/${CURRENT_DIR}/${PIC}
     DEPENDS ${CURRENT_DIR}/${PIC}.fig)
 endforeach()

--- a/manual/reference/programs/CMakeLists.txt
+++ b/manual/reference/programs/CMakeLists.txt
@@ -36,7 +36,7 @@ foreach(PROG ${XTP_SCRIPTS})
       > ${CURRENT_DIR}/${prog}.t2t
     COMMAND ${TXT2TAGS_EXECUTABLE} -t tex --no-header --infile ${CURRENT_DIR}/${prog}.t2t
       --outfile ${CURRENT_DIR}/${prog}.tex
-      DEPENDS ${PROG} ${CURRENT_DIR}/config.t2t)
+      DEPENDS ${prog}_build ${CURRENT_DIR}/config.t2t)
   list(APPEND PROG_TEXS ${CURRENT_DIR}/${prog}.tex)
   file(APPEND ${CMAKE_CURRENT_BINARY_DIR}/${CURRENT_DIR}/all.tex.cmake "\\input{${CURRENT_DIR}/${prog}}\n")
 endforeach()

